frictionless/package/package.py

```diff
@@ -186,6 +190,7 @@ def __init__(
         self.setinitial("name", name)
         self.setinitial("id", id)
         self.setinitial("licenses", licenses)
+        self.setinitial("dialect", dialect)
```

Comment:
Hi @shashigharti, sorry that's in v4 is kinda confusing (will be improved in v5) but can you please store the dialect as `self.__dialect = dialect` the same way as detector (you can see it under the extra comment in the arg list) -- the reason is that dialect is not a part of the data package spec (the same as detector) so we should not store it in the package descriptor

Reply:
The PR looks really well btw :+1: